implementations/rust/ockam/ockam_node/src/storage/database/sqlx_database.rs

```diff
@@ -101,8 +111,13 @@ impl SqlxDatabase {
     }
 
     async fn migrate(&self) -> Result<()> {
+        Self::migrate_tables(&self.pool).await?;
+        self.migrate_attributes_node_name().await
```

Comment:
I have some concerns about this migration strategy. SQL and rust migrations should be paired so we can run/test step-by-step migrations. I haven't looked if this is possible with sqlx, but we could give it a try if we agree on doing something like this.

Reply:
Approach in this PR is from this PR by @etorreborre https://github.com/build-trust/ockam/pull/6979
Pairing the migration would be an improvement, I agree. Not sure how easy it will be to achieve with sqlx